subsys/bluetooth/host/scan.c

```diff
@@ -1335,6 +1335,15 @@ void bt_le_scan_cb_unregister(struct bt_le_scan_cb *cb)
 	sys_slist_find_and_remove(&scan_cbs, &cb->node);
 }
 
+void bt_le_scan_uninit(void)
```

Comment:
considering we're just removing callbacks here, could you update the name to reflect that and be consistent with the other fns you added to remove callbacks?